src/index.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'typeface-roboto';
```

Comment:
Consider adding an empty line between groups of imports.

Reply:
done